proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/client/widget/workflow/WorkflowJobFormView.java

```diff
@@ -413,7 +413,7 @@ public void performAction(ActionEvent event) {
     }
 
     private Widget createMaterialList() {
-        materialView = new WorkflowMaterialView(i18n, true);
+        materialView = new WorkflowMaterialView(i18n, true, this.getClass().getSimpleName());
```

Comment:
Všiml jsem si, že je metoda emulovaná, snad to vrací vždy co má. Proto jsem se ptal. Tohle je dobré pro logy nebo pro tvorbu dočasných identifikátorů. U konfigurace bych dal přednost konstantě, protože pokud někdo třídu přejmenuje nebo kód přesune jinam, tak je to nekompatibilní změna. Konstanta by se hodila i pro případnou implementaci resetu konfigurace. Můžeš to prosím změnit?